website/search/elastic_search.py

```diff
@@ -13,11 +13,10 @@
 from elasticsearch import (
     Elasticsearch,
     RequestError,
-    NotFoundError
+    NotFoundError,
+    ConnectionError
 )
 
-from requests.exceptions import ConnectionError
```

Comment:
Is it possible for a `requests.exceptions.ConnectionError` to be raised? Should we catch both `ConnectionErrors`?